pkgs/tools/networking/dae/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, llvmPackages_15
+, fetchFromGitHub
+, buildGoModule
+}:
+buildGoModule rec {
+  pname = "dae";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "daeuniverse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-EoStRmyYOtvG5ejtvHWNe9IIeE77hqp1OXBhRdxCYHs=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-vxHufE3538l6zIcozFcrNhl+2sG1PtzkVxC0NyL3WMU=";
+
+  proxyVendor = true;
+
+  GOFLAGS = "-buildvcs=false";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/daeuniverse/dae/cmd.Version=${version}"
+    "-X github.com/daeuniverse/dae/common/consts.MaxMatchSetLen_=64"
+  ];
+
+  preBuild = ''
+    unset STRIP
+    make CFLAGS="-D__REMOVE_BPF_PRINTK -fno-stack-protector" \
+    CLANG=${lib.getExe llvmPackages_15.clang} \
```

```suggestion
    CLANG=${lib.getExe buildPackages.clang} \
```
(If the default version of clang works)